jani_generator/src/jani_generator/scxml_helpers/scxml_tags.py

```diff
@@ -205,9 +205,14 @@ def _append_scxml_body_to_jani_automaton(jani_automaton: JaniAutomaton, events_h
                         new_edge.destinations[0]['assignments'].append(JaniAssignment({
                             "ref": f'{param_assign_name}.length',
                             "value": f"{variable_name}.length"}))
+                # TODO: get the expected type from a jani expression, w/o setting dummy def. values
```

Comment:
This is very temporary, and should be addressed in issue https://github.com/convince-project/AS2FM/issues/31